google/cloud/spanner/value_test.cc

```diff
@@ -937,101 +936,124 @@ TEST(Value, CommitTimestamp) {
   EXPECT_FALSE(bad.ok());
 }
 
-class ValueStreamOperator
-    : public TestWithParam<std::tuple<std::string, Value, std::string>> {
- public:
-  static std::string TestName(
-      TestParamInfo<ValueStreamOperator::ParamType> const& info) {
-    return std::get<0>(info.param);
-  }
-};
+TEST(Value, OutputStream) {
+  auto const normal = [](std::ostream& os) -> std::ostream& { return os; };
+  auto const hex = [](std::ostream& os) -> std::ostream& {
+    return os << std::hex;
+  };
+  auto const boolalpha = [](std::ostream& os) -> std::ostream& {
+    return os << std::boolalpha;
+  };
+  auto const float4 = [](std::ostream& os) -> std::ostream& {
+    return os << std::showpoint << std::setprecision(4);
+  };
+  auto const alphahex = [](std::ostream& os) -> std::ostream& {
+    return os << std::boolalpha << std::hex;
+  };
```

Comment:
Possibly
```
  auto const alphahex = [&hex, &boolalpha](std::ostream& os) -> std::ostream& {
    return hex(boolalpha(os));
  };
```
to localize the particular variety of bool and hex formatting to one place?

Reply:
Unless you feel strongly, I think the current code is simpler and clearer and would prefer to keep it.